.github/ubuntu18/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:18.04
+
+# Install updates to base image
+RUN \
+    apt-get update \
+    && apt-get install -y \
+    git \
+    wget \
+    sudo \
+    build-essential \
+    debhelper \
+    dpkg-dev \
+    extra-cmake-modules \
+    libc6-dev \
+    libfontconfig1-dev \
+    libfreetype6-dev \
+    libharfbuzz-dev \
+    libx11-xcb-dev \
+    libyaml-cpp-dev \
+    make \
+    ninja-build \
+    ncurses-bin \
+    pkg-config \
+    libxml2-utils \
+    software-properties-common \
+    libqt5gui5 \
+    libqt5opengl5-dev \
+    libqt5x11extras5-dev \
+    qtbase5-dev \
+    qtmultimedia5-dev
+
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test
+RUN apt-get update
+RUN apt-get install -y g++-10
```

Comment:
I think we can drop this line, meaning, a default compiler should be pulled in as dependency via `install-scripts.sh`.

Reply:
Default compiler is g++-8 and to install g++-10 we need to add-apt-repository, i think it's better to have this here than update install-deps.sh script to handle docker image of ubuntu 18.04

Reply:
Ok. Understood. :)